Monika After Story/game/script-topics.rpy

```diff
@@ -12999,3 +12999,34 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_intrusivethought",category=['Psychology'],prompt="Intrusive thoughts",random=True))
+
+label monika_intrusivethought:
+    m 1euc "Hey [player]..."
+    m 1ekd "Have you ever had thoughts that you find upsetting, distressing, or all around unpleasant?"
```

Comment:
I think this is fine enough as is, but it kind of sounds to me like Monika is about to start a commercial.